### PR TITLE
Feature/add basic UI

### PR DESCRIPTION
**Pull request by daudi13**
- Created an app.rb file that will serve as your console app entry-point. It should have methods that do the following:
  - List all books.
  - List all people.
- Created a person (teacher or student, not a plain Person).
  - Created a book.
  - Created a rental.
  - List all rentals for a given person id.

---
Thank you for taking the time to review my code

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+
+  def list_id_rentals
+    return puts 'Please add a rental first' if @rental.empty?
+
+    puts "\nID of person: "
+    # @people.each do |person|
+    #   puts "ID: #{person.id}, [#{person.class}] Name: #{person.name}, Age: #{person.age}"
+    # end
+    person_id = gets.chomp.to_i
+    rentals = @rental.select { |rent| rent.person.id == person_id }
```
Comment (ShoiraTa) [inline app.rb:175]: - [ ] I can successfully create student, teacher, books rentals. But when I want to list all rentals for a given person id, I am getting an error `undefined method id for #<Book:0x000000010320fe28`
Screenshot 2022-05-20 at 10 47 12

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @daudi13 ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

**First let me to point out:**

- School library console app behaves in the same was as the [example video](https://www.youtube.com/watch?v=vkkgrhD6aXQ) ✔️ 
- Create teacher, student, books rentals options work as expected ✔️ 

Keep on coding, good job so far! :clap:

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my Github username.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 1 above)
Comment (daudi13) [reply app.rb:175]: Thank you @ShoiraTa for pointing that out

**Review by Gambit142 — APPROVED** [review]
Hi @daudi13 👋, 

Great work implementing the changes suggested by a previous reviewer. 👏 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Optional Suggestion

- In the **_Get Started section_** of your `README.md` file, it would be nice to include instructions for starting this console app. **_That is `run ruby main.rb` to start the app_**. This would make it easy for anyone who clones this repository to start interacting with your application. 👍

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by daudi13** [comment]
> Hi @daudi13 wave,
> 
> Great work implementing the changes suggested by a previous reviewer. clap
> 
> Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: Congratulations! tada
> 
> ### Optional Suggestion
> * In the **_Get Started section_** of your `README.md` file, it would be nice to include instructions for starting this console app. **_That is `run ruby main.rb` to start the app_**. This would make it easy for anyone who clones this repository to start interacting with your application. +1
> 
> Cheers and Happy coding!clapclapclap
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Thank you @Gambit142

**Pull request merged**